enginetest/queries/queries.go

```diff
@@ -9559,6 +9637,34 @@ FROM mytable;`,
 		Query:    "select TABLE_NAME, IS_UPDATABLE from information_schema.views where table_schema = 'mydb'",
 		Expected: []sql.Row{{"myview1", "YES"}, {"myview2", "YES"}, {"myview3", "NO"}, {"myview4", "NO"}, {"myview5", "YES"}},
 	},
+	// time to json cast is broken
+	{
+		Query: "SELECT json_type(cast(cast('2001-01-01 12:34:56.123456' as time) as json));",
+		Expected: []sql.Row{
+			{"TIME"},
+		},
+	},
+	// missing year cast
```

Comment:
Are we still missing something for this? I saw we added cast to year syntax in vitess and some code in GMS for casting. Just curious what else is needed for this one.

Reply:
oops this test should not be skipped; will fix